index.html

```diff
@@ -177,41 +177,47 @@ <h4 class="card__title">Skills</h4>
 
   <!-- Contact -->
   <section class="contact">
-      <div class="container">
-          <div class="contact__text">
-              <h2 class="heading-2">I'm always interested in hearing about new projects, so if you'd like to chat, please get in touch.</h2>
-          </div>
-          <div class="contact__form">
-              <form>
-                  <div class="form-control">
-                      <input type="text" placeholder="Full name">
-                  </div>
-                  <div class="form-control">
-                      <input type="email" placeholder="Email Address">
-                  </div>
-                  <div class="form-control">
-                      <textarea name="" id="" cols="30" rows="10" placeholder="Enter text here"></textarea>
-                  </div>
-                  <button class="btn">Get in touch</button>
-              </form>
-          </div>
-      </div>
-  </section>
-
-  <!-- Footer -->
-  <footer class="footer">
-      <div class="containter">
-           <div class="footer__social">
-              <ul class="hero__social">
-                  <li><a href="#" class="hero__social-links"><img src="./images/twitter.svg" alt="Twitter"></a></li>
-                  <li><a href="#" class="hero__social-links"><img src="./images/Linkedin icon.svg" alt="Linkedin"></a></li>
-                  <li><a href="#" class="hero__social-links"><img src="./images/medium.svg" alt="Meduim"></a></li>
-                  <li><a href="#" class="hero__social-links"><img src="./images/github.svg" alt="Github"></a></li>
-                  <li><a href="#" class="hero__social-links"><img src="./images/angellist.svg " alt="Angllist"></a></li>
-              </ul>
-              <a href="#" class="footer__email">dselasea@gmail.com</a>
-           </div>
-      </div>
-  </footer>
+    <div class="container">
+        <div class="contact__text">
+            <h2 class="heading-2">I'm always interested in hearing about new projects, so if you'd like to chat, please get in touch.</h2>
+        </div>
+        <div class="contact__form">
+            <form action="https://formspree.io/f/mrgdrewl"
+            method="POST">
+                <div class="form-control">
+                    <label>
+                        <input name="full_name" type="text" placeholder="Full name" required maxlength="30">
+                    </label>
+                </div>
+                <div class="form-control">
+                    <label>
+                        <input name="email" type="email" placeholder="Email Address" required>
+                    </label>
+                </div>
+                <div class="form-control">
+                    <label>
+                        <textarea name="message" cols="30" rows="10" placeholder="Enter text here" required maxlength="500"></textarea>
```

Comment:
For the input fields please match the design to the Figma Template. You can achieve this by adding the following styles to each input tag including `text area` 

`padding: 0.8rem;
    border-radius: 4px;
    font-family: 'Inter';
    font-style: normal;
    font-weight: 400;
    font-size: 15px;
    line-height: 24px;
    display: flex;
    align-items: center;
    color: #6B778C;`

It will add to your design and will make it looks more professional.